src/Core/Checkout/Cart/Delivery/DeliveryCalculator.php

```diff
@@ -174,7 +174,7 @@ private function matches(Delivery $delivery, ShippingMethodPriceEntity $shipping
         }
 
         // $end (optional) exclusive
-        return ($value >= $start) && (!$end || $value <= $end);
+        return ($value >= $start || abs($value-$start) < 0.00001) && (!$end || $value <= $end || abs($value-$end) < 0.00001);
```

Comment:
Can you use this class?
https://github.com/shopware/platform/blob/trunk/src/Core/Framework/Util/FloatComparator.php#L5

Reply:
Done